### PR TITLE
Testing POA Verified and POA rejected scenario

### PR DESCRIPTION
**Pull request by bertrand-deriv**
### Testing POA Verified and POA rejected scenario
- created tests for the POA verified scenario
- created tests for the POA rejected scenario
- created utility function for date generation

### CODE REVIEW

**Review by markodonnell-deriv — CHANGES_REQUESTED** [review]
Do these tests need to be in separate spec file?
They look quite similar and could be differnet it()'s within the same describe and the common steps could be shared within a function in the same spec file?

**Review by markodonnell-deriv — CHANGES_REQUESTED** [review]
Also, c_createRealAccount set env vars which I've been meaning to review before we use it in e2e tests. I think it might impact other tests.
As you wrote it, would you mind double-checking that?

**Comment by markodonnell-deriv** [comment]
> Also, c_createRealAccount set env vars which I've been meaning to review before we use it in e2e tests. I think it might impact other tests. As you wrote it, would you mind double-checking that?

Disregard this now, as per discussions.

**Review by markodonnell-deriv — APPROVED** [review]
Looks good.

File: cypress/e2e/wip/KYC/poaVerifyAndReject.cy.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
+import '@testing-library/cypress/add-commands'
+import { getCurrentDate } from '../../../support/helper/utility'
+
+const BO_URL = `https://${Cypress.env('configServer')}/d/backoffice/login.cgi`
```
Comment (meeyun-deriv) [inline cypress/e2e/wip/KYC/poaVerifyAndReject.cy.js:4]: put the BO URL in env var as well

**Review by meeyun-deriv — CHANGES_REQUESTED** [review]

File: cypress/e2e/wip/KYC/poaVerifyAndReject.cy.js (hunk 1 above)
Comment (bertrand-deriv) [reply cypress/e2e/wip/KYC/poaVerifyAndReject.cy.js]: I have implemented it

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Pull request merged**